.github/workflows/iroha2.yml

```diff
@@ -51,6 +51,23 @@ jobs:
       - name: Run benchmarks
         run: cargo bench --workspace --verbose
 
+  build-wasm32:
+    needs: [coverage, bench]
+    runs-on: ubuntu-latest
+    working-directory: wasm
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set nightly toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2021-02-20
+          target: wasm32-unknown-unknown
+          override: true
+      - name: Test
+        run: cargo test --tests --release --verbose -- --test-threads=1
```

Comment:
1) Shouldn't this test be in the PR workflow. If this fails, we can't see that it did indeed fail. 
2) We do build the wasm crate, but it isn't saved as an artifact anywhere. Why are we building it then? 
3) This invocation is at odds with all other invocations: some comments (in the yaml file) would be useful:
	3.1) Why is it `cargo test --tests`
	3.2) Does it break with `mold -run cargo test --tests`, and if so, wouldn't it make sense to open an issue for `mold`?
	3.3) Why is this test run in `--release` mode. 
	3.4) Why is this test run in `--verbose` mode, most other tests are run with `--quiet` so that the logs are shorter. 
	3.5) Why `--test-threads=1`. Is there an issue we should be fixing?

Reply:
I just literally copied this line from the job below. Would you like me to make these modifications for that  existing job?

Reply:
We should, but I don't want to over-encumber this PR. Let's merge as is, and optimise later.